07LessonSeven/in-class-project/css/my-style.css

```diff
@@ -0,0 +1,59 @@
+#top{
+  background-image: url("../img/nasa-background.jpg");
+  text-align: center;
+  width: 850px;
+  height: 450px;
+  margin-bottom: 15px;
+}
+
+#top>h1{
+  color: white;
+  font-family: "Arial Black", sans-serif;
+  font-weight: bold;
+  font-size: 100px;
+  line-height: 450px;
+}
+
+#bot{
+  width: 850px;
+}
+
+.info{
```

Comment:
instead of giving it a class, you could have gone through the bot ID element as well